inbox/config.py

```diff
@@ -109,9 +109,16 @@ def _update_config_from_env(config, env):
                 raise
 
 
-def _get_local_feature_flags(config):
-    flags = config.get("FEATURE_FLAGS", "").split()
+def _update_config_from_env_variables(config):
```

Comment:
Note that sync-engine historically only supported configuration using config files but this is awkward with Docker and other tools. It will be possible to override those two things with environment variables now.